index.html

```diff
@@ -15,392 +16,228 @@
+                </ul>
+                <div>
+                    <p class="uniqe-para">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
+                </div>
+                <div class="nav-source">
+                    <ul>
+                        <li>html</li>
+                        <li>Ruby on rails</li>
+                        <li>JavaScript</li>
+                        <li>css</li>
+                    </ul>
+                </div>
+                <div class="button-m">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="back-image-file-3 all-image-pro mid-end"></div>
+        </article>
+    </section>
+    <section class="about-me" id="about-1">
+        <article>
+            <div class="about-me-page">
+                <h1 class="header-1">About Myself</h1>
+                <p>Hello I’m a software developer! I can help you build a  <br> product , feature or website Look through some of my <br>
+                     work and experience! If you like what you see and have <br> a project you need coded, don’t hestiate to contact me.</p>
+                <div class="let-s">
+                    <h2>LET'S CONNECT</h2>
+                    <div class="vector">
+                        <ul>
+                            <li><a href="#" class="twitter"></a></li>
+                            <li><a href="#" class="linkedin"></a></li>
+                            <li><a href="#" class="medium"></a></li>
+                            <li><a href="#" class="github"></a></li>
+                            <li><a href="#" class="hand"></a></li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="button-m">
+                    <button type="button">See Project</button>
+                </div>
+            </div>
+            <div class="lists-of-frame">
+                <ul>
+                    <li class="language"><h1>Language</h1>
+                        <div class="up-down">
+                        <span  id="odd-span"  class="glyphicon glyphicon-chevron-right"></span></p>
+                        <span  id="new-span"  class="glyphicon glyphicon-chevron-down"></span></p>
+                        </div>
+                        
+                    
+                    <ul class="hovers">
+                        <li>
+                            <span class="list-1"></span>
+                             <p>JavaSript</p>
+                        </li>
+                        <li>  
+                            <span class="list-2"></span>                                
+                            <p>Htlm</p>
+                        </li>
+                        <li>
+                            <span class="list-3"></span>                             
+                            <p>Css</p>
+                        </li>
```

Comment:
- [OPTIONAL] Kindly make sure that the mobile version in the section of Languages is rendering correctly after the changes added to make the desktop version. :eyes: :eyes:  Kindly fix this tiny issue. It will help to show your portfolio more professional to recruiters. :briefcase: 
 
Your design:
![image](https://user-images.githubusercontent.com/98363075/194675574-de3898f6-9ae0-4143-a795-49543de49ea4.png)

Figma design:
![image](https://user-images.githubusercontent.com/98363075/194675748-af71b308-e965-4205-9824-0c6550d2e641.png)